packages/executor/src/modules/eth.ts

```diff
@@ -194,12 +194,12 @@ export class Eth {
 
     // calculate callGasLimit based on paid fee
     const { cglMarkup } = this.config;
-    const totalGas: BigNumber = BigNumber.from(paid).div(userOp.maxFeePerGas);
-    let callGasLimit = totalGas
-      .sub(preOpGas)
-      .mul(10000 + this.config.cglMarkupPercent)
-      .div(10000) // % markup
-      .add(cglMarkup || 0);
+    // const totalGas: BigNumber = BigNumber.from(paid).div(userOp.maxFeePerGas);
+    // let callGasLimit = totalGas
+    //   .sub(preOpGas)
+    //   .mul(10000 + this.config.cglMarkupPercent)
```

Comment:
pls retain `cglMarkupPercent`

Reply:
uncommenting the part would totally override the callGasLimit received from Proxy
should i use the the below instead ?
`callGasLimit = callGasLimit.add(cglMarkup || 0);`
@ch4r10t33r @0xSulpiride

Reply:
you should first add `cglMarkupPercentage` (value in bps (10000 = 100%, 5000 = 50%, etc...)) and then add `cglMarkup` on top of it